tests/react/vanilla-utils/atomWithObservable.test.tsx

```diff
@@ -183,47 +183,6 @@ it('writable count state with delayed value', async () => {
   await findByText('count: 9')
 })
 
-it('only subscribe once per atom', async () => {
```

Comment:
This is the requirement and the reason why `atomWithObservable` exists in the Jotai core repo. Can you preserve this behavior?

Reply:
I have updated the test to show that there will be in fact one active subscription per atom.